assets/src/edit-story/components/form/numeric.js

```diff
@@ -37,7 +37,8 @@ import MULTIPLE_VALUE from './multipleValue';
 const DECIMAL_POINT = (1.1).toLocaleString().substring(1, 2);
 
 const StyledInput = styled(Input)`
-  width: 100%;
+  flex: 1 1 auto;
+  width: 0;
```

Comment:
Why is `width=0`  here? What happens with `width=auto`?

Reply:
It's a CSS Hack to make Firefox work with inputs controlled by `display: flex` correctly. `auto` doesn't work, but I replace it by `min-width: 0` to avoid set `width` with non-sense value.